src/react/workflow/replication/Replication.jsx

```diff
@@ -0,0 +1,272 @@
+                            <T.Value>
+                                <Controller
+                                    control={control}
+                                    id='sourcePrefix'
+                                    name='sourcePrefix'
+                                    render={({ onChange, value: sourcePrefix }) => {
+                                        return <Input
+                                            onChange={handleChange(onChange)}
+                                            value={sourcePrefix}
+                                            autoComplete='off'
+                                        />;
+                                    }}
+                                />
+                            </T.Value>
+                        </T.Row>
+                    </T.GroupContent>
+                </T.Group>
+                <T.Group>
+                    <T.GroupName>
+                        Destination
+                    </T.GroupName>
+                    <T.GroupContent>
+                        <T.Row>
+                            <T.Key> Location Name </T.Key>
+                            <T.Value>
+                                <Controller
+                                    control={control}
+                                    id='destinationLocation'
+                                    name='destinationLocation'
+                                    render={({ onChange, value: destinationLocation }) => {
+                                        const options = destinationOptions(locations);
+                                        return <Select
+                                            onChange={handleChange(onChange)}
+                                            options={options}
+                                            formatOptionLabel={renderDestination(locations)}
+                                            value={options.find(l => l.value === destinationLocation.value)}
+                                        />;
+                                    }}
+                                />
+                                <T.ErrorContainer>
+                                    <ErrorInput hasError={errors.destinationLocation?.value}> {errors.destinationLocation?.value?.message} </ErrorInput>
+                                </T.ErrorContainer>
+                            </T.Value>
+                        </T.Row>
+                    </T.GroupContent>
+                </T.Group>
+            </T.Groups>
+            <T.Footer>
+                <Button disabled={!isCreating && !showEditWorkflowNotification} id='cancel-workflow-btn' style={{ marginRight: '8px' }} outlined onClick={handleCancel} text='Cancel'/>
+                <Button disabled={!isCreating && !showEditWorkflowNotification} icon={<i className="fas fa-save" />} id='create-workflow-btn' variant="secondary" onClick={handleSubmit(onSubmit)} text={isCreating ? 'Create' : 'Save'}/>
```

Comment:
According to the design:
```suggestion
                <Button disabled={!isCreating && !showEditWorkflowNotification} icon={<i className="fas fa-save" />} id='create-workflow-btn' variant="secondary" onClick={handleSubmit(onSubmit)} text={isCreating ? 'Create' : 'Save Changes'}/>
```